case-lib/config.sh

```diff
@@ -90,3 +90,13 @@ FALLBACK_TO_PROC=${FALLBACK_TO_PROC:-false}
 
 # Skip the storage checks if they aren't required
 ENABLE_STORAGE_CHECKS=${ENABLE_STORAGE_CHECKS:-0}
+
+# start_test in lib.sh will check whether FW is loaded before every test case.
+# If FW LOADING is not a prerequisite for the test, or if you just want to skip
+# the check, then define NO_POLL_FW_LOADING to true.
+# NO_POLL_FW_LOADING=true
+
+# Set MAX Polling time to check FW Loading. If FW is already loaded, it will
+# return immediately. Default value is set to 60 seconds, because i915 driver
+# timeout is 60 seconds.
+MAX_WAIT_FW_LOADING=${MAX_WAIT_FW_LOADING:-60}
```

Comment:
If i915 driver is 60 seconds, shouldn't this be 65 seconds?

Also: can you please add at least one reference for more context? Maybe some of these:
- https://github.com/thesofproject/sof-test/issues/291
- https://github.com/thesofproject/linux/issues/3042
- https://github.com/thesofproject/sof-test/issues/979
- other?

Reply:
> If i915 driver is 60 seconds, shouldn't this be 65 seconds?

Good point, I think about this too. But there are some other delays already applied top of this. so it should have plenty of room

Reply:
> But there are some other delays already applied top of this

Once we merge this then we should get rid of those.

Reply:
Also: this is only a maximum. The test will start as soon as the firmware has booted.

Reply:
okay, I will update default value to 70. if you plan to remove all extra delays, I need some extra margin